noise.py

```diff
@@ -159,40 +167,40 @@ def _generate_smooth_noise(self, base, octave=None):
         :class:`list[list[float]]`
             The new "smoothed noise".
         """
-        interpolate = self.interpolate
+        interpolate: Callable = self.interpolate
```

Comment:
`Callable` should be hinted -> `Callable[[float, float, float], float]`

Reply:
Fixed.